digitalocean/Droplet.py

```diff
@@ -136,7 +136,7 @@ def create_multiple(*args, **kwargs):
 
         droplets = []
 
-        data = api.get_data("droplets", type=POST, params=data)
+        data = api.get_data('droplets/', type=POST, params=data)
```

Comment:
Is there any specific reason to change all of the double quotes into single quotes? It is hard to review and unnecessary to try to uniform this style only in some places.